common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -316,6 +317,7 @@ class @Problem
       switch response.success
         when 'incorrect', 'correct'
           window.SR.readElts($(response.contents).find('.status'))
+          @el.trigger('contentChanged', [@id, response.contents])
```

Comment:
`this.contents` from `Sequence` module contains the original contents that need to be updated 
– attaching screenshot.

<img width="1252" alt="screen shot 2016-04-04 at 2 07 29 am" src="https://cloud.githubusercontent.com/assets/7848408/14358542/db371204-fd06-11e5-8d7d-aba4e481dc92.png">